app/models/news_article.rb

```diff
@@ -1,6 +1,15 @@
 class NewsArticle < Node
+  has_many :news_distributions
+  has_many :sections, through: :news_distributions, source: :distribution,
+           source_type: 'Section'
+
+  alias_method :publisher, :section
+
   store_attribute :data, :release_date, Date
 
+  content_attribute :short_summary
+  content_attribute :summary
```

Comment:
I think all content needs summary, and it should be versioned so perhaps a node field.